frontend/src/views/account-mgmt/ManageFacilityView.vue

```diff
@@ -168,7 +176,9 @@ export default {
       facility: undefined,
       loading: false,
       rules,
+      editMode: false,
       editModePrimaryContact: false,
+      disableEditPrimaryContact: false,
```

Comment:
I actually don't think this is required. Can't we just disable when editMode = true since EditFacilityContacts.vue and the Primary Contact code all handle this flag now?